checkpointctl.go

```diff
@@ -47,22 +48,28 @@ func setupShow() *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.BoolVar(
-		&printStats,
+		&stats,
 		"print-stats",
 		false,
-		"Print checkpointing statistics if available",
+		"Print checkpointstatistics if available",
```

Comment:
"checkpoint statistics" are two words

Reply:
Ack I think I messed up while rebasing. I'll fix it when I rebase after #61 is merged.